holoviews/operation/datashader.py

```diff
@@ -336,7 +336,24 @@ def _get_agg_params(self, element, x, y, agg_fn, bounds):
 
 
 
-class aggregate(AggregationOperation):
+class LineAggregationOperation(AggregationOperation):
+
+    line_width = param.Number(default=None, bounds=(0, None), doc="""
+        Width of the line to draw, in pixels. If zero, the default,
+        lines are drawn using a simple algorithm with a blocky
+        single-pixel width based on whether the line passes through
+        each pixel or does not. If greater than one, lines are drawn
+        with the specified width using a slower and more complex
+        antialiasing algorithm with fractional values along each edge,
+        so that lines have a more uniform visual appearance across all
+        angles. Line widths between 0 and 1 effectively use a
+        line_width of 1 pixel but with a proportionate reduction in
+        the strength of each pixel, approximating the visual
+        appearance of a subpixel line width.""")
```

Comment:
Looks good though might be worth mentioning a `line_width` of 1 as a special value: the antialiased version of the old default behavior.

Reply:
I also see arguments for the default being `None` (old behavior) or setting the default to `1` (nicer looking lines, need to set it back to None to restore the old behavior exactly).

Reply:
Actually, given that this feature isn't going to be in the datashader version people have right now, it is probably best to stick with `None`. This docstring should mention the datashader version that antialiasing/line_width gets used.

Essentially, people may expect to see antialiasing because they set `line_width` but they have an old datashader version. Now I am leaning towards setting it to 1 after the new datashader release has been out for a while.

Reply:
I'd guess that most people will get an updated Datashader at the same time they will get an updated HoloViews, so though it's good to have error checking I think we should assume the support is available in Datashader when setting defaults.  I don't think `line_width=1` has any special significance we need to mention.

Reply:
That isn't necessarily true unless we want to pin to greater than the new datashader version in the new release (which I am not convinced we should do, especially for a hotfix release).

Reply:
It's not *necessarily* true, but isn't it roughly true? I.e., when someone creates a new environment with HoloViews or hvPlot, seems like they'll get a new Datashader too, since that will also be available and compatible. It's true they could be updating only and precisely HoloViews, but that takes extra work, and seems less likely.